docroot/modules/humsci/hs_layouts/hs_layouts.module

```diff
@@ -41,3 +43,15 @@ function hs_layouts_preprocess_container(&$variables) {
     $variables['#attached']['library'][] = 'hs_layouts/layout_builder_admin';
   }
 }
+
+/**
+ * Implements hook_preprocess_preprocess_pattern_spotlight().
+ */
+function hs_layouts_preprocess_pattern_spotlight(&$variables) {
+  $paragraph = $variables['context']->getProperty('entity');
+  if ($paragraph->hasField('field_hs_spotlight_bg') && !$paragraph->field_hs_spotlight_bg->isEmpty()) {
+    $bgValue = $paragraph->field_hs_spotlight_bg->first()->getValue()['value'];
+
+    $variables['spotlight_bg_class'] = Html::cleanCssIdentifier($bgValue);
```

Comment:
```suggestion
  if ($paragraph->hasField('field_hs_spotlight_bg') && $paragraph->get('field_hs_spotlight_bg')->count()) {
    $bgValue = $paragraph->get('field_hs_spotlight_bg')->getString();

    $variables['attributes']['class'][] = Html::cleanCssIdentifier($bgValue);
```

Reply:
I'm not sure if its `$variables['attributes']` or `$variables['#attributes']` with the `#`.

Using this way, you can remove the class from the twig template because it adds it to the attributes already.

Reply:
Interesting. Thanks for this Mike, I'll take a shot with this.

Reply:
@pookmish I can see it added to the attributes when I dump the variables to the page but it isn't getting showing up in the template. By the way, it did need the `#`.

Reply:
apparently `attributes` was an object, not an array. I've modified my suggestion and it looks to work on my side

Reply:
Working on my side as well. Pushing a commit now.